packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -12,10 +12,16 @@ import { composeRenderFunction } from '@uifabric/utilities';
 
 const getClassNames = classNamesFunction<IChoiceGroupOptionStyleProps, IChoiceGroupOptionStyles>();
 
+const LARGE_IMAGE_SIZE = 71;
+
 /**
  * {@docCategory ChoiceGroup}
  */
-export class ChoiceGroupOptionBase extends React.Component<IChoiceGroupOptionProps, any> {
+export class ChoiceGroupOptionBase extends React.Component<IChoiceGroupOptionProps, {}> {
+  public static defaultProps: Partial<IChoiceGroupOptionProps> = {
+    imageSize: { width: 32, height: 32 },
```

Comment:
Can you add a comment explaining why it's important for this to be in defaultProps?

I wonder how many other places we have similar issues...

Reply:
yes it's on my to-do list to check every where :) but outside of scope of this change